mss/windows.py

```diff
@@ -201,11 +189,31 @@ def close(self):
         except AttributeError:
             pass
 
+    def _set_dpi_awareness(self):
+        """
+        Set DPI aware to capture full screen on Hi-DPI monitors
+        Automatically scale for DPI changes
+        """
+        
+        version = sys.getwindowsversion()[:2]
+        if version >= (6, 3):
+            # Windows 8.1+
+            # Here 2 = PROCESS_PER_MONITOR_DPI_AWARE, which means:
+            #     per monitor DPI aware. This app checks for the DPI when it is
+            #     created and adjusts the scale factor whenever the DPI changes.
+            #     These applications are not automatically scaled by the system.
+            ctypes.windll.shcore.SetProcessDpiAwareness(2)
+        elif (6,0) <= version < (6, 3):
+            # Windows Vista, 7, 8 and Server 2012
+            self.user32.SetProcessDPIAware()
+
     @property
     def monitors(self):
         # type: () -> Monitors
         """ Get positions of monitors (see parent class). """
 
+        self._set_dpi_awareness()
```

Comment:
Shouldn't we move this call inside `__init__()`?

Reply:
ok